client/src/component/signin/index.jsx

```diff
@@ -0,0 +1,128 @@
+import { React, useState } from 'react';
+import {
+  Layout,
+  Row,
+  Col,
+  Image,
+  Form,
+  Button,
+  Input,
+  message,
+} from 'antd';
+
+import axios from 'axios';
+import 'antd/dist/antd.css';
+import './index.css';
+
+const {
+  Content,
+} = Layout;
+
+function Signin() {
+  const [userData, setUserData] = useState({ email: '', password: '' });
+
+  const handleInputChange = (event) => {
+    const { name, value } = event.target;
+    setUserData({
+      ...userData,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    try {
+      // Post data here
+      await axios.post('/api/signin', userData);
+      message.success('Welcome back!');
+    } catch (e) {
+      message.warning('Try again');
+    }
+  };
+
+  return (
+    <Layout>
+      {/* --- Navbar --- */}
+      <Content>
+        <Row
+          justify="end"
+        >
+          <Col span={12} className="login_image">
+            <Image
+              style={{ height: '100vh', width: '100vh' }}
```

Comment:
move the style to your style files

Reply:
Some components didn't work when i try to override their styles

Reply:
You can check the antd class, and change its style.